src/App.js

```diff
@@ -1,19 +1,77 @@
-import React from 'react';
+import React, { Component } from 'react';
 import './App.css';
 
 import users from './api/users';
+import todoList from './api/todos';
+import TodoList from './components/TodoList/TodoList';
+import NewTodo from './components/NewTodo/NewTodo';
 
-function App() {
-  return (
-    <div className="App">
-      <h1>Add todo form</h1>
-
-      <p>
-        <span>Users: </span>
-        {users.length}
-      </p>
-    </div>
-  );
+class App extends Component {
+  state = {
+    todoList: [...todoList]
+      .sort((a, b) => b.id - a.id)
+      .map(todo => ({
+        ...todo,
+        user: users.find(user => user.id === todo.userId),
+      })),
+  }
+
+  NewTodo = (newTask, userId) => {
```

Comment:
```suggestion
  addTodo = (newTask, userId) => {
```
методы называем с маленькой буквы, я вообще сразу не понял как это работает, подумал что ты в Компонент NewTodo передаешь его же)

Reply:
И вообще это все должно храниться в компоненте newTodo. В App должен остаться только метод, который тут в стейте добавляет todo в todoList, он не должен знать ни о каких statusToggle.

Reply:
Я оставил в App только метод для добавления нового todo, но свойства id и user все равно пришлось оставить